packages/account/src/Components/poinc/file-uploader-container/file-uploader-container.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { Icon } from '@deriv/components';
+import { isDesktop, WS } from '@deriv/shared';
+import { Localize } from '@deriv/translations';
+import FileUploader from 'Components/file-uploader';
+
+const PoincFileUploaderContainer = ({ document_type, is_description_enabled = true, getSocket, onFileDrop, onRef }) => {
+    const ref = React.useRef();
+    const getSocketFunc = getSocket ?? WS.getSocket;
+
+    React.useEffect(() => {
+        if (ref && typeof onRef === 'function') onRef(ref);
+        return () => onRef(undefined);
+    }, [onRef, ref]);
+
+    const properties = [
+        { name: 'size', icon: 'IcLessThanEight', text: 'Less than 8MB' },
+        { name: 'format', icon: 'IcImage', text: 'JPEG JPG PNG PDF' },
+        { name: 'with-address', icon: 'IcUser', text: 'Issued under your name with your current address' },
+        { name: 'time', icon: 'IcClock', text: '1 - 6 months old' },
+    ];
+
+    return (
+        <div className='account__file-uploader-section'>
+            {is_description_enabled && (
+                <ul className='account__file-uploader-list'>
+                    {properties.map(({ name, icon, text }) => (
+                        <li key={name} className='account__file-uploader-box'>
+                            <Icon icon={icon} className='account__file-uploader-icon' size={24} />
+                            <div className='account__file-uploader-item'>
+                                <Localize i18n_default_text={text} />
```

Comment:
component <Localize /> works with variables.  we should not pass variables into localize()

Reply:
I don't think Localize component works with variables. The string has to be passed as `i18n_default_text ` prop and should be in the quotes. In the current implementation, the strings won't be fetched to crowdin and it will not be translated.